packages/charrua-core/charrua-core.0.1/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+name: "charrua-core"
+version: "0.1"
+maintainer: "Christiano F. Haesbaert <haesbaert@haesbaert.org>"
+authors: "Christiano F. Haesbaert <haesbaert@haesbaert.org>"
+homepage: "https://github.com/haesbaert/charrua-core"
+bug-reports: "https://github.com/haesbaert/charrua-core/issues"
+license: "ISC"
+dev-repo: "https://github.com/haesbaert/charrua-core.git"
+available: [ocaml-version >= "4.02"]
+build: [
+  ["sh" "build.sh"]
+]
+depends: [
```

Comment:
Here are also a few comments about your build procedure: 

1. You are relying on a static `.install` file for installation. This will break for example in switches where only the byte-code compilers are available. Also you don't have support for native dynamic linking (`.cmxs` files) in there. 
2. You are using a shell script which is Windows unfriendly. 

You can use [`topkg`](https://github.com/dbuenzli/topkg) to solve these problems, it shouldn't change anything to your current setup, except add a new file and change your package build procedure.